packages/augur-core/src/contracts/symbiote/SymbioteHatchery.sol

```diff
@@ -0,0 +1,143 @@
+pragma solidity 0.5.15;
+pragma experimental ABIEncoderV2;
+
+import 'ROOT/symbiote/ISymbioteShareTokenFactory.sol';
+import 'ROOT/symbiote/ISymbioteShareToken.sol';
+import 'ROOT/symbiote/IArbiter.sol';
+import 'ROOT/libraries/Initializable.sol';
+import 'ROOT/para/interfaces/IParaUniverse.sol';
+import 'ROOT/para/interfaces/IParaOICash.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+contract SymbioteHatchery is Initializable {
+    using SafeMathUint256 for uint256;
+
+    uint256 private constant MIN_OUTCOMES = 2; // Does not Include Invalid
+    uint256 private constant MAX_OUTCOMES = 7; // Does not Include Invalid
+    uint256 private constant MAX_FEE = 2 * 10**16; // 2%
+    address private constant NULL_ADDRESS = address(0);
+    uint256 private constant MAX_UINT = 2**256 - 1;
+
+    struct Symbiote {
+        address creator;
+        uint256 creatorFee;
+        uint256 numTicks;
+        IArbiter arbiter;
+        ISymbioteShareToken[] shareTokens;
+        uint256 creatorFees;
+    }
+
+    Symbiote[] public symbiotes;
+    ISymbioteShareTokenFactory public tokenFactory;
+    IParaUniverse public paraUniverse;
+    IParaOICash public OICash;
+    IFeePot public feePot;
+    IERC20 public underlyingCurrency;
+
+    event SymbioteCreated();
+    event CompleteSetsMinted();
+    event CompleteSetsBurned();
+    event Claim();
+    event AugurResolutionInitiated();
```

Comment:
Why no additional information in the logs?

Reply:
Ah, I was leaving these empty but as suggestions till I knew what we think we'll want. Should've commented. I'll put in some basic data for now.